### PR TITLE
COCICCS-2325 MetaData Creation for SSL certificate

### PR DESCRIPTION
**Pull request by Ukothur1**

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java [hunk 1]
```diff
@@ -459,15 +484,148 @@ public CertManagerLogin login(CertManagerLoginRequest certManagerLoginRequest) t
...
+        }
+
+        CertificateData certDetails = null;
+        //Get Certificate Details
+        for (int i = 1; i <= retrycount; i++) {
+            Thread.sleep(delayTime);
+            certDetails = getCertificate(sslCertificateRequest, certManagerLogin);
+            log.debug(JSONUtil.getJSON(ImmutableMap.<String, String>builder().
+                    put(LogMessage.USER, ThreadLocalContext.getCurrentMap().get(LogMessage.USER).toString()).
+                    put(LogMessage.ACTION, String.format("Fetching certificate details count = [%s] and status = [%s]"
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:526]: LogMessage.ACTION is different from LogMessage.MESSAGE. They have to be correctly used...

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java [hunk 2]
```diff
@@ -459,15 +484,148 @@ public CertManagerLogin login(CertManagerLoginRequest certManagerLoginRequest) t
...
+        return ControllerUtil.convetToJson(rqstParams);
+    }
+
+
+    /**
+     * get Application Details
+     * @param api
+     * @return
+     */
+    public  JsonObject getApplicationDetails(String api)  {
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:571]: Is this different from WorkloadDetailsService.getApiResponse(...)?

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java (hunk 1 above)
Comment (Ukothur1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:526]: Corrected

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java (hunk 2 above)
Comment (Ukothur1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:571]: Yes Siva, Slightly different, as part of this change I am fetching information only for the selected applications. But in another place, we are fetching all applications and we need to fetch more information than this . That why I created a separate method to fulfill this requirement and if i  change that method, sure there will be  required testing and same method using in UI as well

**Pull request merged**